lasagne/layers/base.py

```diff
@@ -300,3 +307,50 @@ def get_output_for(self, inputs, **kwargs):
         `NotImplementedError`.
         """
         raise NotImplementedError
+
+
+class UnLayer(MergeLayer):
+
+    """
+    lasagne.layers.UnLayer(incoming, layer, **kwargs)
+
+    The :class:`UnLayer` class performing inverse operations
+    for a single layer of a neural network: transposed layer
+    for a :class:`DenseLayer`, deconvolutional layer for
+    :class:`Conv2DLayer`, :class:`Conv1DLayer`; or
+    an unpooling layer for :class:`MaxPool2DLayer`.
+
+    It is specially useful for building (convolutional)
+    autoencoders with tied parameters.
+
+    Parameters
+    ----------
+    incoming : a :class:`Layer` instance
+        The layer feeding into this layer.
```

Comment:
That's incomplete. Please copy from any of the other layers (e.g., from `Layer` above).